molvis/molvisgl/src/processors/molecularrasterizer.cpp

```diff
@@ -165,11 +224,15 @@ void MolecularRasterizer::process() {
         }
     }();
 
-    if (meshes_.empty() || inport_.isChanged() || updateColorMap) {
+    if (meshes_.empty() || inport_.isChanged() || updateColorMap || brushing_.isChanged()) {
         meshes_.clear();
+        const size_t pickingId = atomPicking_.getPickingId(0);
+        size_t offset = 0;
         for (auto structure : inport_) {
-            meshes_.push_back(
-                createMesh(*structure, {coloring_, atomColormap_, aminoColormap_, fixedColor_}, 0));
+            meshes_.push_back(createMesh(*structure,
```

Comment:
Would it be safe to reuse the meshes, since they are internal to the rasterization? or would that still present issues?

Reply:
I mean that you should just update the existing mesh when the brushing changes for example, not recreate all of it